spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -217,6 +217,18 @@ def paginated_collection(*args)
           info = pagination.find_by_class('pagination_information').first.content.gsub('&nbsp;', ' ')
           expect(info).to eq "Displaying posts <b>1 - 30</b>"
         end
+
+        it "should not perform count queries" do
+          Post.create!
+          collection = Post.page(1).per(30)
+          log = StringIO.new
+
+          expect(collection).not_to receive(:total_pages)
+          ActiveRecord::Base.logger = Logger.new(log)
+          paginated_collection(collection, pagination_total: false)
+          ActiveRecord::Base.logger = nil
+          expect(log.string).to_not include 'SELECT COUNT'
+        end
```

Comment:
Since the fix is not implemented yet, this spec fails:

```

  1) ActiveAdmin::Views::PaginatedCollection creating with the dsl with :pagination_total set to false should not perform count queries
     Failure/Error: end
       expected "D, [2018-04-02T22:48:21.957729 #82970] DEBUG -- :   \e[1m\e[35m (0.2ms)\e[0m  \e[1m\e[34mSELECT COUN...T  1 FROM \"posts\" LIMIT ? OFFSET ?) subquery_for_count\e[0m  [[\"LIMIT\", 30], [\"OFFSET\", 0]]\n" not to include "SELECT COUNT"
```

and the log output shows the count queries:

```

D, [2018-04-02T22:48:21.957729 #82970] DEBUG -- :    (0.2ms)  SELECT COUNT(*) FROM (SELECT  1 FROM "posts" LIMIT ? OFFSET ?) subquery_for_count  [["LIMIT", 1], ["OFFSET", 0]]
D, [2018-04-02T22:48:22.172004 #82970] DEBUG -- :    (0.2ms)  SELECT COUNT(*) FROM (SELECT  1 FROM "posts" LIMIT ? OFFSET ?) subquery_for_count  [["LIMIT", 30], ["OFFSET", 0]]
D, [2018-04-02T22:48:22.172614 #82970] DEBUG -- :   Post Load (0.1ms)  SELECT  "posts".* FROM "posts" ORDER BY "posts"."id" ASC LIMIT ? OFFSET ?  [["LIMIT", 1], ["OFFSET", 0]]
D, [2018-04-02T22:48:22.175192 #82970] DEBUG -- :    (0.1ms)  SELECT COUNT(*) FROM (SELECT  1 FROM "posts" LIMIT ? OFFSET ?) subquery_for_count  [["LIMIT", 30], ["OFFSET", 0]]
D, [2018-04-02T22:48:22.175794 #82970] DEBUG -- :    (0.1ms)  SELECT COUNT(*) FROM (SELECT  1 FROM "posts" LIMIT ? OFFSET ?) subquery_for_count  [["LIMIT", 30], ["OFFSET", 0]]
```